tests/integration/components/bs-modal-test.js

```diff
@@ -127,6 +127,31 @@ module('Integration | Component | bs-modal', function (hooks) {
     assert.dom('.modal').hasAttribute('aria-labelledby', modalTitleId);
   });
 
+  test('it allows to render modal in fullscreen using @fullscreen argument', async function (assert) {
```

Comment:
As that feature is supported in Bootstrap 5 only, we should also only run the test for BS5. We have an utility function called `testBS5` to do so: https://github.com/kaliber5/ember-bootstrap/blob/4b261eba2d6352eafdade589e4ba2a5a9f3e750a/tests/helpers/bootstrap.js#L26-L28 You could find an usage example here: https://github.com/kaliber5/ember-bootstrap/blob/a254a1ff8de05f0cc6fa5d51bbb92d3ad827b5ce/tests/integration/components/bs-button-group-test.js#L54

Reply:
Fixed